e2e/pages/manageCases/caseProgression/completeTheOrder/adminEditAndApproveAnOrder1Page.ts

```diff
@@ -0,0 +1,59 @@
+import { Page } from "@playwright/test";
+// import AccessibilityTestHelper from "../../../../common/accessibilityTestHelper";
+import { Selectors } from "../../../../common/selectors";
+import { AdminEditAndApproveAnOrder1Content } from "../../../../fixtures/manageCases/caseProgression/completeTheOrder/adminEditAndApproveAnOrder1Content";
+import { Helpers } from "../../../../common/helpers";
+import { CommonStaticText } from "../../../../common/commonStaticText";
+
+export class AdminEditAndApproveAnOrder1Page {
+  public static async adminEditAndApproveAnOrder1Page(
+    page: Page,
+    accessibilityTest: boolean,
+  ): Promise<void> {
+    await this.checkPageLoads(page, accessibilityTest);
+    await this.fillInFields(page);
+    await this.continue(page);
+  }
+
+  private static async checkPageLoads(
+    page: Page,
+    accessibilityTest: boolean,
+  ): Promise<void> {
+    await page
+      .locator(`${Selectors.GovukHeadingL}`, {
+        hasText: `${AdminEditAndApproveAnOrder1Content.govUkHeadingL}`,
+      })
+      .waitFor();
+    await Promise.all([
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.GovukFormLabel}:text-is("${AdminEditAndApproveAnOrder1Content.formLabel1}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.button}:text-is("${CommonStaticText.previous}")`,
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.button}:text-is("${CommonStaticText.continue}")`,
+        1,
+      ),
+    ]);
+    if (accessibilityTest) {
+      // await AccessibilityTestHelper.run(page);
+    }
+    // #TODO commented out until ticket resolved FPET-1224
+  }
+
+  private static async fillInFields(page: Page): Promise<void> {
+    await page.selectOption("#draftOrdersDynamicList", { index: 1 });
```

Comment:
On this case I used index because it only has one option so to select hence would only be index 1 and also since the dropdown value is dynamic so wasn't sure how to select using text/label